test/service.ts

```diff
@@ -17,9 +17,12 @@
 import * as assert from 'assert';
 import * as extend from 'extend';
 import * as proxyquire from 'proxyquire';
+import {Service, ServiceConfig} from '../src/service';
```

Comment:
Maybe make this `import {Service as ServiceClass}` and then later `const Service: typeof ServiceClass = proxyquire('...')`